streampark-flink/streampark-flink-proxy/src/main/scala/org/apache/streampark/flink/proxy/FlinkShimsProxy.scala

```diff
@@ -36,7 +36,7 @@ object FlinkShimsProxy extends Logger {
   )
 
   private[this] val SHIMS_PATTERN = Pattern.compile(
-    "streampark-flink-shims_flink-(1.12|1.13|1.14|1.15)_(2.11|2.12)-(.*).jar",
+    "streampark-flink-shims_flink-(1.12|1.13|1.14|1.15|1.16)_(2.11|2.12)-(.*).jar",
```

Comment:
Hi @lvshaokang , thanks for your contribution. Most look good.

Too many pattern, please extract a const field.

Reply:
> Too many pattern, please extract a const field.

Hi, @1996fanrui . What you mean is that extract regex pattern to a const field?
such this `streampark-flink-shims_flink-(1.12|1.13|1.14|1.15|1.16)_(2.11|2.12)-(.*).jar, (^|.*)streampark-flink-shims_flink-(1.12|1.13|1.14|1.15|1.16)-(.*).jar$`